odk1-src/form-styling.rst

```diff
@@ -164,15 +156,36 @@ you'll need to escape that too.
   :header: type, name, label, hint
   
   note, escape_md, \# This headline is normal sized, \*Asterisks\* and \_underscores\_ and one slash: \\
+
+.. _inline-html:
+
+Inline HTML
+~~~~~~~~~~~~~
+
+Many Markdown implementations support inline HTML,
+but Collect only supports a small subset of HTML elements.
+Support of HTML is further limited because:
+
+- Your exact Android device, operating system version,
+  and other device-related factors
+  will affect what HTML can be rendered, and how it is rendered.
+- HTML is not supported
+  by other form rendering tools in the XForm ecosystem.
+  For example, HTML elements that work in Collect may not work in Enketo.
+
+For these reasons, we do not recommend using HTML in forms (except the ``<span>`` element :ref:`noted below <custom-styling>`).
+
+.. seealso:: `The list of HTML tags currently supported in Collect <https://www.grokkingandroid.com/android-quick-tip-formatting-text-with-html-fromhtml>`_.
+
   
 .. _custom-styling:
```

Comment:
I think that this should go under Markdown in the ToC, not between Markdown and Emoji as it is currently. @lognaturel thinks it's fine. What do you think, @adammichaelwood?

Reply:
I wrote the text thinking it was under Markdown, 
I just did the wrong headline underline.

Reply:
I don't think of the `<span>` construct as being a part of Markdown support,
but I don't have a strong opinion on the matter.

Reply:
It does occur to me that this would be more clear if it was "Custom font styling"

Reply:
Maybe then the text at the very top of the document can read, "Labels, hints, and choices in an XLSForm can all be styled using Markdown, fonts and colors, and emoji." Then we change the title to "Fonts and colors"?

Reply:
That is, form styling is already custom styling, so it's a bit redundant to call it that.

Reply:
I like this suggestion.